diff --git a/node_modules/bopit/src/body.js b/node_modules/bopit/src/body.js
index 4a7c7cd..7f58fa8 100644
--- a/node_modules/bopit/src/body.js
+++ b/node_modules/bopit/src/body.js
@@ -87,7 +87,7 @@ class Body {
     return this.world.queryRect(x, y, w, h)
       .map((other) => this._collide(other, goalX, goalY, filter))
       .filter((collision) => !!collision)
-      .sort((a, b) => a.distance < b.distance)
+      .sort((a, b) => a.distance - b.distance)
   }
 
   _collide (other, goalX, goalY, filter = (o) => defaultResp) {
